continuum/scenarios/segmentation.py

```diff
@@ -0,0 +1,220 @@
+import warnings
+from copy import copy
+from typing import Callable, List, Union, Optional
+import os
+import multiprocessing
+
+import numpy as np
+from PIL import Image
+import torchvision
+
+from continuum.datasets import _ContinuumDataset
+from continuum.scenarios import ClassIncremental
+from continuum.tasks import TaskSet
+from continuum.download import ProgressBar
+
+
+class SegmentationClassIncremental(ClassIncremental):
+    """Continual Loader, generating datasets for the consecutive tasks.
+
+    Scenario: Each new tasks bring new classes only
+
+    :param cl_dataset: A continual dataset.
+    :param nb_tasks: The scenario number of tasks.
+    :param increment: Either number of classes per task (e.g. increment=2),
+                    or a list specifying for every task the amount of new classes
+                     (e.g. increment=[5,1,1,1,1]).
+    :param initial_increment: A different task size applied only for the first task.
+                              Desactivated if `increment` is a list.
+    :param transformations: A list of transformations applied to all tasks.
+    :param class_order: An optional custom class order, used for NC.
+                        e.g. [0,1,2,3,4,5,6,7,8,9] or [5,2,4,1,8,6,7,9,0,3]
```

Comment:
ce commentaire n'est pas 'a jour

Reply:
Yes, doc is not yet updated, see my todo list in the description of the PR